docs/artifacts.md

```diff
@@ -0,0 +1,38 @@
+# Artifacts
+
+Each individual input arguments and output value of Sematic functions are
+serialized, persisted, and tracked by Sematic.
+
+For example, for the following function
+
+```python
+@sematic.func
+def foobar(foo: Foo, bar: Bar) -> Output:
+    # business logic
+    return output
+```
+
+three artifacts will be created:
+
+* One for the input argument `foo`
+* One for the input argument `bar`
+* One for the output value `output`
+
+## Accessing artifacts
+
+In order to access an artifacts values from outside your pipeline, find the artifact you want to access in the UI, and click on its artifact ID to copy it:
+
+![Artifact ID](images/artifactId.png)
+
+
+then simply do:
+
+```python
+from sematic import get_artifact_value
```

Comment:
I think it would be helpful to put the user-facing api calls under a single module. So users can do something like `from sematic import client`  or `from sematic import api` then `client.get_artifact_value(...)`. It would keep the top level namespace from getting *too* polluted, and also make it a bit more clear to folks when they're doing something that needs to talk to a server. I do agree it should be a subset of the contents of `api_client`, though.